packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -317,18 +315,18 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
 
       if (calendars && calendars.length > 0) {
         calendars.map(async (cal) => {
-          await prisma.selectedCalendar.delete({
+          await prisma.selectedCalendar.deleteMany({
```

Comment:
Something like this: 
```suggestion
         prisma.selectedCalendar.delete({
              userId_integration_externalId: {
                userId: user.id,
                externalId: cal.externalId,
                integration: cal.integration as string,
              },
            },
        }).catch(async catching in another thread);
```